newclient/src/modules/uncles/pages/PageDetailsUncle.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+    <v-container grid-list-lg class="mb-0">
+        <app-bread-crumbs :new-items="crumbs" />
+        <v-layout row wrap justify-start class="mb-4">
+            <v-flex xs12>
+                <app-details-list :title="title" :details="uncleDetails" :is-loading="loading" :error="error" />
+            </v-flex>
+        </v-layout>
+    </v-container>
+</template>
+
+<script lang="ts">
+import AppBreadCrumbs from '@app/core/components/ui/AppBreadCrumbs.vue'
+import AppDetailsList from '@app/core/components/ui/AppDetailsList.vue'
+import { eth } from '@app/core/helper'
+import { Detail, Crumb } from '@app/core/components/props'
+import { Vue, Component, Prop } from 'vue-property-decorator'
+import { getUncleByHash } from '@app/modules/uncles/handlers/uncleDetails.graphql'
+import { UncleDetails as UncleDetailsType } from '@app/modules/uncles/handlers/UncleDetails.type'
+
+@Component({
+    components: {
+        AppBreadCrumbs,
+        AppDetailsList
+    },
+    apollo: {
+        uncleDetail: {
+            query() {
+                const self = this as any
```

Comment:
Is this from  the old files?
can you look at BlockDetailsPage and Block Details Handler? and follow similar structure.